client/ayon_traypublisher/plugins/create/create_editorial.py

```diff
@@ -843,7 +843,7 @@ def get_pre_create_attr_defs(self):
                 ],
                 allow_sequences=False,
                 single_item=False,
-                label="Sequence file",
+                label="Edit list file",
```

Comment:
While we're at it - would it make sense to add a tooltip to these attributes explaining what the Edit list file is?

Reply:
yeah, sure!

Reply:
BTW what is "Edit list file"? Is it like "Editorial file"?

Reply:
> BTW what is "Edit list file"? Is it like "Editorial file"?

It's like EDL (Edit decision list) file.

Reply:
So the users using it will understand?

Reply:
yes, the file def mentions EDL file extension and also the YT video mentions that.
![image](https://github.com/user-attachments/assets/ab1cf978-c2cb-4956-8239-de6e0950a27a)

https://www.youtube.com/watch?v=oSDskPRINHU

Reply:
I've added the tool tip here
https://github.com/ynput/ayon-traypublisher/pull/47/commits/9f01495673f58730030f41d465fc34f769dd2d44